app/assets/stylesheets/components/navbar.scss

```diff
@@ -92,7 +93,8 @@
 
 @media (min-width: 768px) and (max-width: 1024px)  {
   .navbar{
-    padding: 0px 30px 0px 30px;
+    padding: 12px 30px 0px 30px;
+    margin-bottom: 62px;
```

Comment:
I'm not sure if there is a good enough reason to include these styles? 

The logo should sit snug to the top of the screen as designed the 50px of margin the navbar has by default is that size to fit the visual hierarchy of the spacing.

Reply:
This was needed for some obscure reason to match the previous position when the divider was ruling. But not needed if we revert that